SpiffWorkflow/bpmn/specs/BpmnProcessSpec.py

```diff
@@ -68,6 +70,41 @@ def deserialize(self, serializer, wf_spec, s_state):
         return serializer.deserialize_join(wf_spec, s_state, _EndJoin)
 
 
+class BpmnDataSpecification:
+
+    def __init__(self, name, description=None):
+        """
+        :param name: the name of the task (the BPMN ID)
+        :param description: the task description (the BPMN name)
+        """
+        self.name = name
+        self.description = description or name
+        # In the future, we can add schemas defining the objects here.
+
+    def get(self, my_task):
+        """Copy a value form the workflow data to the task data."""
+        if self.name not in my_task.workflow.data:
+            raise WorkflowDataException(my_task, data_input=self)
+        my_task.data[self.name] = deepcopy(my_task.workflow.data[self.name])
+
+    def set(self, my_task):
```

Comment:
it might make it more readable for some of these method names (get, set, and copy) to be longer, from my perspective.